activesupport/lib/active_support/inflector/inflections.rb

```diff
@@ -1,4 +1,6 @@
-require 'concurrent'
+# FIXME: clean up warning spam from concurrent lib
```

Comment:
I think they are already fixed. Can we just point to master branch of it just to avoid having patches on Rails?